recipes/python-dp/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
 
 test:
```

Comment:
Just wanted to clarify, what does this test command do?

Reply:
it imports the package.

Reply:
it already imports the package right? shouldn't we have it run the test suite instead?

Reply:
it is failing to test importing the package, after building, which would have to done in order for the package to be used.